examples/visualization/brain.py

```diff
@@ -0,0 +1,106 @@
+"""
+.. _ex-brain:
+
+===============================
+Plotting with ``mne.viz.Brain``
+===============================
+
+In this example, we'll show how to use :class:`mne.viz.Brain`.
+"""
+# Author: Alex Rockhill <aprockhill@mailbox.org>
+#
+# License: BSD-3-Clause
+
+# %%
+# Plot a brain image
+# ------------------
+#
+# First we plot a brain. This requires a directory with MRI? information.
+# MNE provides functionality to automatically retrieve standard head images.
+# In this example we use the ``sample`` data which is data from an example
+# subject being shown auditory and visual stimuli.
+
+import os.path as op
+import matplotlib.pyplot as plt
+
+import mne
+from mne.datasets import sample
+
+print(__doc__)
+
+data_path = sample.data_path()
+subjects_dir = op.join(data_path, 'subjects')
+brain = mne.viz.Brain('sample', subjects_dir=subjects_dir,
+                      alpha=0.5, background='white', cortex='low_contrast')
+
+
+# %%
+# Add source information
+# ----------------------
+#
+# Plot source information.
+sample_dir = op.join(data_path, 'MEG', 'sample')
+evoked = mne.read_evokeds(op.join(sample_dir, 'sample_audvis-ave.fif'))[0]
+trans = mne.read_trans(op.join(sample_dir, 'sample_audvis_raw-trans.fif'))
+stc = mne.read_source_estimate(op.join(sample_dir, 'sample_audvis-meg'))
+stc.crop(0.09, 0.1)
+
+fmin, fmax = stc.data.min(), stc.data.max()
+for i, hemi in enumerate(('lh', 'rh')):
+    hemi_data = stc.data[:len(stc.vertices[i])]
+    hemi_vertices = stc.vertices[i]
+    brain.add_data(hemi_data, fmin=fmin, hemi=hemi, fmax=fmax,
+                   vertices=hemi_vertices, alpha=0.5,
+                   smoothing_steps='nearest', time=stc.times)
+
+
+# %%
+# Modify the view of the brain
+# ----------------------------
+#
+# You can adjust the view of the brain using ``show_view`` method.
+brain.show_view(azimuth=190, elevation=70, distance=500)
+
+
+# %%
+# Highlight a region on the brain
+# -------------------------------
+#
+# It can be useful to highlight a region of the brain for analyses.
+# To highlight a region on the brain you can use the ``add_label`` method.
+# Labels are stored in the Freesurfer label directory for that subject.
+# Here we will show Brodmann Area 44.
```

Comment:
Are all Brodman areas available? Or just a subset?

Reply:
In this example just a few because it's the sample data and no one wants all the bells and whistles and having to wait 15 minutes for it to download. In general for your own freesurfer recon, they should all be there I think

Reply:
I think we need to explicitly say this then. As a user the first thing I did was run this example and change it to Brodman 41 and it errors.

I've been using MNE for a year now and have no experience with freesurfer. I don't think knowledge of freesurfer should be assumed. And if it's required for using Brain then we should add the required explanation or links here.

Reply:
I.e the average user will just try and use their favourite Bradman area. When it doesn't work they will assume it's an MNE error. Not some complex freesurfer usage issue

If it's not expected to work then we need to spell that out in the tutorial.